content/providers/01-ai-sdk-providers/11-google-vertex.mdx

```diff
@@ -294,6 +294,13 @@ The following optional settings are available for Google Vertex models:
 
   Optional. When enabled, the model will [use Google search to ground the response](https://cloud.google.com/vertex-ai/generative-ai/docs/grounding/overview).
 
+- **audioTimestamp** _boolean_
+
+  Optional. Enables timestamp understanding for audio files. Defaults to false.
+
+  This is useful for generating transcripts with accurate timestamps.
+  Only available for `gemini-1.5-pro-002` and `gemini-1.5-flash-002`.
```

Comment:
Same comment as below re: is there a Vertex doc page we can link to rather than specifying the models here, just to simplify maintenance.